### PR TITLE
[China Network] Add tile

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-3940.

Missing the final tile icon, tracked in DES-4704. We're moving forward with a temporary icon.

### CODE REVIEW

File: content/china-network/_index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  title: Cloudflare China Network
+---
+
+# Cloudflare China Network
+
+The [Cloudflare China Network](https://www.cloudflare.com/china-network/) is a package of selected Cloudflare’s performance and security products running on data centers located in mainland China and operated by Cloudflare's partner JD Cloud.
+
+The data centers cover most populated regions in China. Combining Cloudflare’s technological leadership and JD Cloud’s local operations expertise, the Cloudflare China Network is designed to meet the needs for secure, reliable, and fast-performing content delivery in China. You can use the same configurations that you use with Cloudflare everywhere else in the world and manage your global services from a single dashboard.
+
+The Cloudflare China Network is available as a separate subscription for customers on an Enterprise plan.
```
Comment (kodster28) [inline content/china-network/_index.md]: Might break this into a separate section under the header `Availability`

File: content/china-network/_index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Main features
+
+The Cloudflare China Network provides:
+
+* A single solution for both performance improvement and security services such as WAF, DDoS, and bot management.
```
Comment (kodster28) [inline content/china-network/_index.md]:
```suggestion
* A single solution for both performance improvement and security services such as [WAF](/waf/), [DDoS](/ddos-protection/), and [bot management](/bots/).
```

File: content/china-network/concepts/china-dns.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+weight: 4
+---
+
+# China Authoritative DNS
+
+Cloudflare is able to deploy DNS service in mainland China to improve the Time to First Byte (TTFB) performance. With this deployment, the DNS query will be resolved in mainland China instead of global DNS servers.
+
+## When to use
+
+Before you enable China Authoritative DNS, you should identify that over 90% of the traffic is coming from China. After you enable China Authoritative DNS, all the global DNS requests will be routed to Cloudflare's servers in China.
```
Comment (kodster28) [inline content/china-network/concepts/china-dns.md]:
```suggestion
Before you enable China Authoritative DNS, you should confirm that over 90% of the traffic is coming from China. After you enable China Authoritative DNS, all the global DNS requests will be routed to Cloudflare's servers in China.
```

File: content/china-network/get-started.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Get started
+pcx-content-type: how-to
```
Comment (kodster28) [inline content/china-network/get-started.md]:
```suggestion
pcx-content-type: get-started
```

File: content/china-network/get-started.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+weight: 2
+meta:
```
Comment (kodster28) [inline content/china-network/get-started.md]: I think you're actually fine w/o this. The product name is auto-appended to all page names, so it's already `Get started - Cloudflare China Network`

File: content/china-network/reference/supported-products.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+# Supported products and features
+
+The Cloudflare China Network supports the following products and features:
+
+## Application services
+
+| Product | Feature | Description |
+|---|---|---|
+| DNS | DNS resolution service | DNS resolution inside mainland China. |
```
Comment (kodster28) [inline content/china-network/reference/supported-products.md]: Should include links here to individual product tiles.

**Review by kodster28 — APPROVED** [review]

File: content/china-network/get-started.md (hunk 5 above)
Comment (pedrosousa) [reply content/china-network/get-started.md]: Yes, I'll remove it.

**Pull request merged**